topomodelx/nn/cell/can_bis.py

```diff
@@ -0,0 +1,141 @@
+"""CAN class."""
+
+import torch
+import torch.nn.functional as F
+
+from topomodelx.nn.cell.can_layer import CANLayer, MultiHeadLiftLayer, PoolLayer
+
+
+class CAN(torch.nn.Module):
```

Comment:
this was is supposed to be deleted because it is not faithful to the original implementation and we have a better one in the repo. I am deleting it in the next push.

Reply:
Sounds good